src/commands/speak.ts

```diff
@@ -0,0 +1,15 @@
+import { Message } from 'discord.js'
+
+export async function speak(message: Message) {
+    console.log('here')
+    const random = Math.random()
+    if (random < 0.001) {
+        await message.channel.send('DEATH TO MY ENEMIES.')
+    } else if (random < 0.3) {
+        await message.channel.send('Purrrrr.')
+    } else if (random < 0.6) {
+        await message.channel.send('Mrow.')
+    } else if (random < 1) {
+        await message.channel.send('Meow.')
+    }
```

Comment:
Duplicated code here and in the randomMeow function. Can probably get rid of randomMeow just call this

Reply:
Although we need to think about the 0.05 chance which isn't accounted for here